README.md

```diff
@@ -7,5 +7,75 @@ Check out our integration docs [here](https://dash.applovin.com/documentation/me
 ## Downloads
 See [pub.dev](https://pub.dev/packages/applovin_max) for the latest releases of the plugin.
 
+## Demo App Instructions
+To get started with the demo app, please ensure Flutter is installed on your system. Once everything is properly installed, follow the instructions below to get the demo application up and running. 
+
+1. Update the `SDK_KEY` and ad unit IDs in the `main.dart` file [here](https://github.com/AppLovin/AppLovin-MAX-Flutter/blob/73c5a281439791f4b81f595ab795f6b99cbc9457/applovin_max/example/lib/main.dart#L28). 
```

Comment:
We should specify where to "get" the SDK key, not "set" it. For SDK key, easiest way would probably be to link to the docs here:
https://dash.applovin.com/documentation/mediation/flutter/getting-started/integration#initialize-the-sdk

For ad unit IDs, we can link to this: https://dash.applovin.com/o/mediation/ad_units

Reply:
My bad. updated.